x-pack/plugins/synthetics/server/synthetics_service/formatters/public_formatters/http.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ConfigKey, HTTPFields } from '../../../../common/runtime_types';
+import { Formatter, commonFormatters } from './common';
+import { tlsFormatters } from './tls';
+import { arrayFormatter, objectFormatter } from './formatting_utils';
+import {
+  arrayToJsonFormatter,
+  objectToJsonFormatter,
+  stringToJsonFormatter,
+} from '../private_formatters/formatting_utils';
+
+export type HTTPFormatMap = Record<keyof HTTPFields, Formatter>;
+export const httpFormatters: HTTPFormatMap = {
+  [ConfigKey.MAX_REDIRECTS]: null,
+  [ConfigKey.REQUEST_METHOD_CHECK]: null,
+  [ConfigKey.RESPONSE_BODY_INDEX]: null,
+  [ConfigKey.RESPONSE_HEADERS_INDEX]: null,
+  [ConfigKey.METADATA]: objectToJsonFormatter,
+  [ConfigKey.URLS]: stringToJsonFormatter,
```

Comment:
I'm pretty sure none of these stringToJsonFormatters should be included in the public formatters. I believe that's what caused the quotation marks in the results.

Reply:
I'm pretty sure nothing should be using a `ToJSON` formatter for public formatters.